components/enedis/actions/get-address/get-address.mjs

```diff
@@ -0,0 +1,23 @@
+import enedis from "../../enedis.app.mjs";
+
+export default {
+  type: "action",
+  key: "enedis-get-address",
+  version: "0.0.1",
+  name: "Get address",
```

Comment:
Most words should be capitalized in the action names - "Get Address" or, in a different example, "Get an Address" would also be ok. The same applies to the other actions submitted.
```suggestion
  name: "Get Address",
```